src/components/pages/Theme/Theme.jsx

```diff
@@ -139,6 +139,12 @@ export const Theme = () => {
             Disabled
           </Header>
           <Checkbox label='Disabled Checkbox' disabled />
+
+          <Header size='small' dividing>
+            Form
+          </Header>
+          <Form.Checkbox label='Form checkbox' />
```

Comment:
I think `<Form.Checkbox>` and `<Form.Field control={Checkbox}>` are the same as <Checkbox>`, so I don't think we need all of them. Let's just stick with the `<Checkbox>` for now.

Reply:
I just wanted to specifically show that styles applied to all variants regardless because we're not sure which approach they choose. I don't have separate styles for `<Form.Checkbox>`, so it doesn't really matter if it's there or not

Reply:
Ok, then let's just add a vertical space between the two.